### PR TITLE
Add .travis.yml file

### PR DESCRIPTION
**Pull request by saptaktakalkar**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - "{ echo '[defaults]'; echo 'roles_path = ../'; } >> ansible.cfg"
+  # Install required ansible roles
+  - sudo ansible-galaxy install -r tests/requirements.yml
+
+script:
+# Check the role/playbook's syntax.
+  - ansible-playbook -i tests/inventory tests/test.yml --syntax-check
+
+  # Run the role/playbook with ansible-playbook.
+  - ansible-playbook -i tests/inventory tests/test.yml --connection=local --sudo
```
Comment (abelboldu) [inline .travis.yml]: this is not the travis template we use, please check the other roles and add the multi container travis file, plus all the files required under tests/

Comment (saptaktakalkar) [inline .travis.yml]: Ok

**Comment by saptaktakalkar** [comment]
image

**Comment by saptaktakalkar** [comment]
@abelboldu Added container based travis file and tested it. Please check the attached screenshot above.

File: tests/test_travis.yml [hunk 2]
```diff
@@ -0,0 +1,140 @@
+---
```
Comment (saptaktakalkar) [inline tests/test_travis.yml:1]: Do we really need this file?

Comment (abelboldu) [inline tests/test_travis.yml:1]: this is not needed if same test.yml works for both Vagrant and travis

Comment (saptaktakalkar) [inline tests/test_travis.yml:1]: Removed here: https://github.com/openstack-ansible-galaxy/openstack-heat/pull/3

**Pull request merged**